apps/storybook/stories/design/Icon.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { Icon } from "@kvib/react/src/icon";
+import { HStack, Spacer } from "@kvib/react/src/layout";
+import { Meta, StoryObj } from "@storybook/react";
+import { colors } from "@kvib/react/src/theme/tokens";
+
+const meta: Meta<typeof Icon> = {
+  title: "Design Fundament/Ikoner**",
+  component: Icon,
+  parameters: {
+    docs: {
+      story: { inline: true },
+      canvas: { sourceState: "shown" },
+    },
+  },
+  argTypes: {
+    color: {
+      control: "color",
+      defaultValue: { summary: "black" },
+    },
+    weight: {
+      control: { type: "range", min: 100, max: 700, step: 100 },
+      defaultValue: { summary: 600 },
+    },
+    grade: {
+      control: "radio",
+      options: [-25, 0, 200],
+      defaultValue: { summary: 0 },
+    },
+    size: {
+      control: "radio",
+      options: [20, 24, 40, 48],
+      defaultValue: { summary: 24 },
+    },
+    isFilled: {
+      control: "boolean",
+      defaultValue: { summary: false },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Icon>;
+
+export const defaultIcon: Story = {
+  args: { icon: "favorite" },
+  render: (args) => <Icon {...args} />,
+};
+
+export const exampleIcon: Story = {
```

Comment:
fin måte å sette opp på, men kanskje få med eksempel på alle props? Grade og weight mangler, tenker det er viktigere å vise dette enn flere typer farger og ikoner. Mulig bruk samme type ikon for alle eksemplene slik at en lettere ser forksjell på¨ulike verdier av propsene